### ISSUE TITLE
Caption styling colors not working in Chrome.

### ISSUE THREAD

**Issue by 4getit** [jwplayer/jwplayer#2274]
Caption styling colors not working in Chrome.

Go to https://developer.jwplayer.com/jw-player/demos/customization/captions-styling/

Changing text color and background color isn't working.

Text color defaults to black. Background color doesn't change.

The last version it was working properly was 7.10.6

**Comment by robwalch** [comment]
Hi @4getit,

What version of Chrome are you using on which OS?

I am unable to reproduce in Chrome 60.0.3112.101 on macOS Sierra 10.12.5. I've set the text and background colors, and the captions reflect those changes:

![screen shot 2017-08-25 at 2 13 26 pm](https://user-images.githubusercontent.com/333258/29726964-f20549fc-899f-11e7-901a-eeaf5db788e6.png)

**Comment by 4getit** [comment]
Hi @robwalch ,

Since I'm using the same versions, I figured it was something else.

It seems that since 7.11, canvas fingerprinting is used for something. 

I had extension CanvasFingerprintBlock 1.5 (https://chrome.google.com/webstore/detail/canvasfingerprintblock/ipmjngkmngdcdpmgmiebdmfbkcecdndc) and that blocks the attempt. This caused the caption styling to go all weird.

I've changed the extension to Canvas Blocker (https://chrome.google.com/webstore/detail/canvas-defender/obdbgnebcljmgkoljcdddaopadkifnpm) and instead of blocking, that one returns a random bogus hash. The styling now works.

Especially with privacy concerns and browsers that prevent these kind of things, maybe it's good to see if there is some alternative to fingerprinting?
